cmake/wgrib2/wgrib2-config.cmake

```diff
@@ -0,0 +1,21 @@
+find_path (WGRIB2_INCLUDES
+  wgrib2api.mod
+  HINTS $ENV{wgrib2_ROOT}/include)
+
+find_library (WGRIB2_LIBRARIES
+  names libwgrib2.a
```

Comment:
What about the `libwgrib2_api.a`?
Does this package not provide that?

Reply:
Good point. I was debating on whether to include it or not since I'm not sure what it does. UFS_UTILS builds fine without it, and I think that's the only application that uses it. I'll go ahead and add it because why not.

Reply:
I have included libwgrib2_api